lib/css/CssParser.js

```diff
@@ -169,6 +171,13 @@ class CssParser extends Parser {
 		}
 
 		const module = state.module;
+		if (
+			module.type === CSS_MODULE_TYPE_AUTO &&
+			IS_MODULES.test(module.resourceResolveData.path)
```

Comment:
Why not `module.resourcePath`?

Reply:
I can't find `resourcePath` property on NormalModule or Module.

Reply:
Or should I use `parseResource(module.resource).path` here? It looks like `parseResource` is used elsewhere to get the module's path, for example https://github.com/webpack/webpack/blob/f921a5baca9c7d200eee8204fafaaaf36b7c1451/lib/NodeStuffPlugin.js#L138-L139

Reply:
Yeah, good catch, it is cachable, so don't worry about perf